docs/install_debian.md

```diff
@@ -0,0 +1,94 @@
+# How to install Lean and mathlib on Debian/Ubuntu
+
+This document explains how to get started with Lean and mathlib if you
+are using a Linux distribution derived from Debian (Debian itself,
+Ubuntu, LMDE,...). This document has three parts: installing Lean,
+creating a new project, and working on an existing project.
+
+If you get stuck, please come to [the chat room](https://leanprover.zulipchat.com/) to ask for assistance.
+
+## Installing Lean and mathlib
+
+Here we will discuss the fast way, assuming a lot of trust from you.
```

Comment:
I think it would be good if this section contained a brief list of executables that this one-liner will install on the system.